prooph/event-store-symfony-bundle/0.10/src/Command/CreateEventStreamCommand.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use Prooph\EventStore\EventStore;
+use Prooph\EventStore\Stream;
+use Prooph\EventStore\StreamName;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CreateEventStreamCommand extends ContainerAwareCommand
```

Comment:
This class should probably be part of the bundle, not the recipe, right?

Reply:
Not necessarily. Bundle helps to connect Symfony application with EventStore which can be managed somewhere else. This class is an example of how it can be done and is based on fixed data due most likely it is modified after bundle is installed. Making it part of a bundle is possible however it would require some changes in bundle itself.

Reply:
I can plan those changes and eventually remove this class from the recipe. Although, this PR main focus is to fix the initial configuration which is, at the moment, not entirely correct after the bundle is installed.

Reply:
Let's remove this class for now then. I suppose there is some docs about this anyway.